desktop/apps/consignments2/test/components.js

```diff
@@ -0,0 +1,126 @@
+import * as actions from '../client/actions'
+import StepMarker from '../components/step_marker'
+import SubmissionFlow from '../components/submission_flow'
+import React from 'react'
+import createReactClass from 'create-react-class'
+import reducers from '../client/reducers'
+import { Provider } from 'react-redux'
+import { createStore } from 'redux'
+import { shallow } from 'enzyme'
+
+describe('React components', () => {
+  let initialStore
+
+  beforeEach(() => {
+    initialStore = createStore(reducers)
+  })
+
+  describe('StepMarker', () => {
+    describe('non-logged-in user', () => {
+      it('displays four steps', () => {
+        const wrapper = shallow(
+          <Provider><StepMarker store={initialStore} /></Provider>
+        )
+        const rendered = wrapper.render()
+        rendered.find('.consignments2-step-marker__dot').length.should.eql(4)
+      })
+      
+      it('updates the color of step labels', () => {
+        const wrapper = shallow(
+          <Provider><StepMarker store={initialStore} /></Provider>
+        )
+        const rendered = wrapper.render()
+        rendered.find('.consignments2-step-marker__step.active').length.should.eql(1)
+        const activeText = rendered.text()
+        activeText.should.containEql('Create Account')
+      })
+    })
+  })
+
+  describe('SubmissionFlow', () => {
+    let steps
+    let seededStore
+
+    beforeEach(() => {
+      SubmissionFlow.__Rewire__('StepMarker', createReactClass({ render: () => <div className='step-marker'></div> }))
```

Comment:
I don't think you need `create-react-class` for this -- can use a simple stateless function and remove all references to the render call later on:

```javascript
SubmissionFlow.__Rewire__('StepMarker', () => <div className='step-marker'></div> )
```